tfx/components/transform/executor.py

```diff
@@ -279,11 +279,15 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
           should contain two splits 'train' and 'eval'.
         - schema: A list of type `standard_artifacts.Schema` which should
           contain a single schema artifact.
+        - cache_output_path: Optinal cache input of 'tf.Transform', where cached
```

Comment:
Is it possible to handle this TODO now as well?
https://github.com/tensorflow/tfx/blob/deb618730dc7675c8e9dc75e03b8ca795d49653d/tfx/components/transform/executor.py#L1352
This would likely be very inefficient.

Reply:
Hi, Zohar, could you be more specific about this, do you mean we shouldn't copy this?

btw, we haven't enable rolling range for tfx oss transform (next PR, we just enabled multiple output artifact in this [pr](https://github.com/tensorflow/tfx/pull/2160))

Reply:
No, I meant that maybe we can see if there's a more performant way of copying on platforms where TFX runs that we can take advantage (for example, snapshot) instead of the naive implementation that is currently used (io_utils.copy_dir).
We can also try to optimize this later when we actually have rolling ranges and we can see being a bottleneck.

Reply:
will revisit this once I've submitted the PR for rolling ranges and looked at the performance of this.